app/static/src/app/apiService.ts

```diff
@@ -147,7 +147,7 @@ export class APIService<S extends string, E extends string> implements API<S, E>
     }
 
     private _fullUrl(url: string) {
-        return `${this._baseUrl}${url}`;
+        return this._baseUrl ? `${this._baseUrl}${url}` : url;
```

Comment:
The changes made to baseUrl affected this branch which caused null to be added to the path where baseUrl is empty` http://localhost:3000/apps/day1/null/odin/versions`

So I added this fix, I doubt if this is appropriate.

Reply:
This isn't going to work on the deployed instance, which will need the base url to successfully make requests. I think this has probably become a problem because the request to get the odin api versions is happening before the base url gets set on `Initialise`. I'd suggest moving the dispatch of the action to fetch versions out of the `mounted` of the component and into the `Initialise` action, here: https://github.com/mrc-ide/wodin/blob/main/app/static/src/app/store/appState/actions.ts#L42
The dispatch can happen any time after the base url is set in:
`commit(AppStateMutation.SetApp, { appName, baseUrl });`

Reply:
Why is it working locally and on CI ?

If it's not gonna work on deployment, what will change ?

Reply:
As I understood the problem, header does not have baseUrl which gets initialised. In fact , if baseUrl was given we would have` http://localhost:3000/apps/day1/apps/day1/odin/versions`, duplicating `apps/day1`. 

Even when url is manually constructed ` http://localhost:3000/apps/day1/odin/versions` still won't work. I am very confused.

Reply:
Works with ` <version-menu v-if="initialised" :wodin-version="wodinVersion"></version-menu>`